### PR TITLE
Fix session actions

### LINKED ISSUE #1035: Edit Menu Item On Session Does Not Do Anything
Actually an error is now returned that there is no action handler for the edit menu item.

`there is 1 error <ilios@component:ilios-sessions-list::ember35704> had no action handler for: edit`

This functionality needs to be implemented.

### PR DESCRIPTION
**Pull request by jrjohnson**
Edit menu replaced with icon.
Added and hooked up delete icon.

Fixes #1035

### CODE REVIEW

File: app/locales/es/translations.js [hunk 1]
```diff
@@ -224,6 +224,7 @@ export default {
     'instructors': 'Instructores',
     'instructorsManageTitle': 'Administrar Instructores',
     'confirmObjectiveRemoval': "¿Está seguro que desea eliminar este objetivo?",
+    'confirmSessionRemoval': "¿Está seguro que desea eliminar este sessión?",
```
Comment (jrjohnson) [inline app/locales/es/translations.js]: @dartajax please confirm this translation for `Are you sure you want to delete this session?`

File: app/locales/fr/translations.js [hunk 2]
```diff
@@ -224,6 +224,7 @@ export default {
     'instructors': "Instructeurs",
     'instructorsManageTitle': "Manager Instructeurs",
     'confirmObjectiveRemoval': "Êtes vous sûr de vouloir supprimer cet objectif?",
+    'confirmSessionRemoval': 'Êtes vous sûr de vouloir supprimer cet séance?',
```
Comment (jrjohnson) [inline app/locales/fr/translations.js]: @saschaben please confirm this translation for `Are you sure you want to delete this session?`

Comment (saschaben) [inline app/locales/fr/translations.js]: should be modified to "`Êtes vous sûr de vouloir supprimer cette séance?'`"

**Comment by jrjohnson** [comment]
@saschaben this is ready for review, I just need to confirm the Spanish translation before it is merged.

**Comment by saschaben** [comment]
appears to be A-OK. I'd still like more distance between the edit and delete icons globally, but that's an issue for another moment.

**Comment by saschaben** [comment]
:+1:

File: app/locales/es/translations.js (hunk 1 above)
Comment (saschaben) [inline app/locales/es/translations.js]: in lieu of @dartajax having to wok at night... I think this needs to be modified to:

`¿Está seguro que desea eliminar esta sesión?`

Comment (dartajax) [inline app/locales/es/translations.js]: That translation was very nice.  Thank you.

**Pull request merged**